org.eclipse.scanning.api/src/org/eclipse/scanning/api/scan/models/ScanModel.java

```diff
@@ -216,22 +224,42 @@ public void setDetectors(IRunnableDevice<?>... detectors) {
 		this.detectors = Arrays.asList(detectors);
 	}
 
-	public List<IScannable<?>> getMonitors() {
-		if (monitors == null) {
+	public List<IScannable<?>> getMonitorsPerPoint() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)